exampleCourse/questions/exampleCodeOutput/question.html

```diff
@@ -0,0 +1,19 @@
+<pl-question-panel>
+  <p>
+    The <code>pl-code-output</code> element allows you to easily format the values of parameters.
+  </p>
+  <p>
```

Comment:
Please provide a paragraph at the top describing the element. For each feature, please try to block them using 

```html
<div class="card my-2">
  <div class="card-body">
    <pl-question-panel>
      <p>Some details about what feature the element is displaying.</p>
      <pl-element></pl-element>
    </pl-question-panel>
  </div>
</div>
```

Sample implementation question would be `examplesNumberInput` c.f. 

https://github.com/PrairieLearn/PrairieLearn/blob/a8c431440408dc5cce6a934cc1043a5860201319/exampleCourse/questions/examplesNumberInput/question.html#L1-L19